renderer/main.js

```diff
@@ -486,13 +483,15 @@ function setupIpc () {
   ipcRenderer.on('wt-poster', (e, ...args) => torrentPosterSaved(...args))
   ipcRenderer.on('wt-audio-metadata', (e, ...args) => torrentAudioMetadata(...args))
   ipcRenderer.on('wt-server-running', (e, ...args) => torrentServerRunning(...args))
+
+  ipcRenderer.send('ipcReady')
```

Comment:
This shouldn't matter,  right? Sending the `ipcReady` event and adding all those event handlers all happens on the same tick either way